### DISCUSSION TITLE
OP3 not gathering stats for my podcast?

### DISCUSSION THREAD

**Discussion by kuba-orlik** [skymethod/op3#25] [Q&A] [answered]
We're running a podcast on Castopod (here's the RSS: https://podcast.internet-czas-dzialac.pl/@icd/feed.xml). 

I've confirmed using https://op3.dev/setup that the prefixes are present:

![](https://github.com/skymethod/op3/assets/2697916/08d9d093-d36c-4a69-884d-03a11b384ff6)

However, the stats seem to be empty - https://op3.dev/show/245e140b-fdbf-5c61-853b-404c5014c4c7

![image](https://github.com/skymethod/op3/assets/2697916/ce8a7df4-24b5-4fa9-9f1d-95980c28dbfa)

I know we have listeners, as we're receiving feedback and I myself downloaded and listened to some recent episodes using Antennapod.

Is there something more that we should enable to enable the stats?

**Comment by johnspurlock-skymethod** [comment, marked as answer]
The stats for that show are over here: https://op3.dev/show/dc72a9d596ec4a7c8266ec6aa5ce3823

Probably due to the fact that you have two public feeds with the same content (other one being https://podcast.midline.pl/feed.xml)

**Reply by kuba-orlik** [reply]
Thanks! We'll manage this mess of multiple rss feeds that is the result of migrating from a different platform.
